manifests/e/EaseUS/EaseUSTodoPCTrans/13.2/EaseUS.EaseUSTodoPCTrans.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: EaseUS.EaseUSTodoPCTrans

```

```suggestion
PackageIdentifier: EaseUS.TodoPCTrans
```
Should follow a similar format to current EaseUS packages